src/Tizen.Multimedia.Remoting/WebRTC/MediaFileSource.cs

```diff
@@ -42,8 +40,7 @@ public sealed class MediaFileSource : MediaSource
         /// <param name="type">The <see cref="MediaType"/> of file source.</param>
         /// <param name="path">The file path.</param>
         /// <exception cref="ArgumentNullException"><paramref name="path"/> is null.</exception>
```

Comment:
what about other possible issues: file does not exist, lack of privileges, file contains bad data? are they reported somewhere else, or during playback?

Reply:
It will be check when file source is attached to WebRTC instance. - WebRTC.AddSource method